src/third_party_libs/googlemock/CMakeLists.txt

```diff
@@ -90,9 +90,9 @@ ms_target_include_system_dirs(gmock PUBLIC ${PROJECT_SOURCE_DIR}/include PRIVATE
 target_link_libraries(gmock gtest)
 if(MSVC)
```

Comment:
Could you remove this entire `if(MSVC)` block please?  If it gets left in, it looks like we had to clear these variables for some reason.  Same for the similar block in the gtest CMakeLists.txt.